scripts/test.sh

```diff
@@ -8,7 +8,7 @@ mkdir .cover/ .test/
 trap "rm -rf .test/" EXIT
 
 for pkg in `go list ./... | grep -v /vendor/`; do
-    go test -v -covermode=atomic \
+    go test -race -v -covermode=atomic \
```

Comment:
you'll need this set so the tests run!
```suggestion
    CGO_ENABLED=1 go test -race -v -covermode=atomic \
```